File: 프로그래머스/내일은 코딩테스트 /01. 문자열 다루기/문자열 계산하기.py
def solution(my_string):
    my_string = my_string.split(" ")
    answer = int(my_string[0])
    for s in range(1, len(my_string), 2):
        op = my_string[s]; num = int(my_string[s+1]) 
        if op == '+':
            answer += num
        else:
            answer -= num
    
    return answer

File: 프로그래머스/내일은 코딩테스트 /01. 문자열 다루기/문자열 계산하기.js
function solution(my_string) {
    const exp = my_string.split(" ");
    let answer = parseInt(exp[0]);
    
    for(let i = 1; i < exp.length; i = i + 2){
        const op = exp[i];
        if(op == '+'){
            answer = answer + parseInt(exp[i+1]);
        } else {
            answer = answer - parseInt(exp[i+1]);
        }
    }
    
    return answer;
}
